docs/api/websocket.md

```diff
@@ -525,3 +525,41 @@ If an error occurs, the `success` key in the `result` message will be set to `fa
    }
 }
 ```
+
+When calling a web service and in case an exception occurs, error `code` key can be one of the following:
+
+- `id_reuse`: The request ID was reused.
+- `invalid_format`: An `vol.Invalid` was raised.
+- `not_found`: The requested service was not found.
+- `not_supported`: The requested action is not supported.
+- `unknown_command`: An unknown command was used.
+- `unknown_error`: An unknown error occurred.
+- `unauthorized`: The request was not authorized.
+- `timeout`: The request was timed out.
+- `template_error`: A template error occurred.
+
+When an integration raises an exception, `code` will be one of the following:
```

Comment:
This is only during service calls, ie for that websocket command. Integration specific websocket commands may have other error codes.